Source/ACE.Server/Command/Handlers/AdminShardCommands.cs

```diff
@@ -179,5 +180,19 @@ public static void HandleHelp(Session session, params string[] parameters)
             else if (close)
                 WorldManager.Close(session == null ? null : session.Player, bootPlayers);
         }
+
+        // purgeorphans
+        [CommandHandler("purgeorphans", AccessLevel.Admin, CommandHandlerFlag.None, 0,
+            "Purges all orphaned objects found in the database")]
+        public static void HandlePurgeOrphans(Session session, params string[] parameters)
```

Comment:
Any type of db purge should be done at startup, or, completely separate from ACE, while ACE is offline.